vmdk_plugin/Makefile

```diff
@@ -200,7 +200,8 @@ FPM_COMMON := -p $(BIN) \
 .PHONY: deb
 deb: pkg-prep deb-pkg-prep
 	@$(CHECK) pkg
-	$(FPM) --deb-no-default-config-files $(FPM_COMMON) -d '$(DOCKER_PACKAGE) > $(MIN_DOCKER_VERSION)' -t deb .
+	#$(FPM) --deb-no-default-config-files $(FPM_COMMON) -d '$(DOCKER_PACKAGE) > $(MIN_DOCKER_VERSION)' -t deb .
```

Comment:
I think the cleanup does not have to happen in this PR. We just need it working first; cleanup wil require more testing and I am not sure it will have any impact on the end result.